.github/workflows/send-updates.yml

```diff
@@ -23,8 +23,8 @@ jobs:
       - name: Login as github actions bot
         run: |
           git config --global --add safe.directory $GITHUB_WORKSPACE
-          git config --global user.name 'github-actions[bot]'
-          git config --global user.email 'github-actions[bot]@users.noreply.github.com'
+          git config --local user.email "itcrtrainingnetwork@gmail.com"
```

Comment:
It's good jhudsl-robot has access to OCS  but because I changed this part it wasn't actually getting access @carriewright11